common/config/rush/pnpm-lock.yaml

```diff
@@ -10851,7 +10852,6 @@ packages:
 
   /classnames/2.3.2:
     resolution: {integrity: sha512-CSbhY4cFEJRe6/GQzIk5qXZ4Jeg5pcsP7b5peFSDpffpe1cqjASH/n9UTjBwOp6XpMSTwQ8Za2K5V02ueA7Tmw==}
-    dev: false
```

Comment:
can you run `rush update` again and see if this goes away? I assume this is left over from when classnames was devDependency